docs/Searching.md

```diff
@@ -50,6 +50,12 @@ some kind of *payload* (above, `"zen.2457644.%.xx.HH.uvc"`) that gives a piece
 of search information. (There are some clauses that don’t technically *need* a
 payload, but JSON syntax demands that you put something.)
 
+**NOTE**: You *cannot* have two clauses with the same name in the same query.
+This is a limitation of the JSON format. For instance, you cannot have
+`"name-matches": "%.xx.uv"` and `"name-matches": "%.yy.uv"` in the example
+above. In that case only the last clause is used. Such a search would require
+two separate queries.
+
```

Comment:
Well ... the search syntax supports `and` and `or` clauses ([docs](https://github.com/HERA-Team/librarian/blob/master/docs/Searching.md#or)), so in an emergency you can do something truly gross like:

```
"or": {
  "and": { "name-like": "%.xx.uv" },
  "or": { "name-like": "%.yy.uv" }
}
```

which I think would work. Obviously that is pretty gross.

More helpfully, the docs could point to [#42](https://github.com/HERA-Team/librarian/issues/42), which I think would be the most sensible way to make things easier here.

Reply:
Done.